interfaces/Config/templates/config_switches.tmpl

```diff
@@ -269,11 +269,6 @@
                     </div>
                     <!--#end if#-->
                     <div class="field-pair">
-                        <label class="config" for="enable_meta">$T('opt-enable_meta')</label>
```

Comment:
Why move this option to Specials?
Shouldn't it be something the user is aware of?